app/app.js

```diff
@@ -18,6 +18,7 @@ const getContainerId = () => {
 };
 
 let start = async () => {
+    await timeout(jobConfig.delayRunnerMs);
```

Comment:
maybe include here an `if (jobConfig.delayRunnerMs)`? i think it might be more understandable to the reader, so they don't think we always do a timeout at the beg of each run

Reply:
done